src/test/java/com/kodilla/ecommercee/domain/OrderTestSuite.java

```diff
@@ -1,8 +1,9 @@
-package com.kodilla.ecommercee.domain;
+package com.kodilla.ecommercee.repository;
 
-import com.kodilla.ecommercee.repository.CartRepository;
-import com.kodilla.ecommercee.repository.OrderRepository;
-import com.kodilla.ecommercee.repository.UserRepository;
+import com.kodilla.ecommercee.domain.Cart;
+import com.kodilla.ecommercee.domain.Order;
```

Comment:
ogólnie przyczepiam się tylko do używania get() z optionala, bo z tego co wiem nie chroni przed nullPointer... i lepiej go nie uzywać. Ale to tak tylko po to, żeby się do czegoś przyczepić :)